libr/core/cmd.c

```diff
@@ -970,7 +970,7 @@ R_API int sdbshell_history_up(RLine *line) {
 		return false;
 	}
 	line->sdbshell_hist_iter = line->sdbshell_hist_iter->n;
-	strncpy (line->buffer.data, line->sdbshell_hist_iter->data, R_LINE_BUFSIZE - 1);
+	R_STR_CPY (line->buffer.data, line->sdbshell_hist_iter->data);
```

Comment:
why not just use r_str_ncpy? i dont see the macro helping in here

Reply:
It saves a `sizeof `. I don't have strong opinion if this is helpful